Emittery.d.ts

```diff
@@ -76,6 +76,16 @@ declare class Emittery {
 	 * specified.
 	 */
 	listenerCount(eventName?: string): number;
+
+	/**
+	 * Bind a `methodNames` or all `Emittery` methods if `methodNames` not defined into `target` object.
```

Comment:
```suggestion
	 * Bind the given `methodNames`, or all `Emittery` methods if `methodNames` is not defined, into the `target` object.
```

Reply:
Can you include a usage example.

https://github.com/sindresorhus/typescript-definition-style-guide#documentation